spec/lib/video_info/providers/youtube_spec.rb

```diff
@@ -192,9 +192,13 @@
       end
 
       describe '#author_url' do
-        subject { super().author_url }
         author_url = 'https://www.youtube.com/channel/UCzxQk-rZGowoqMBKxGD5jSA'
-        it { is_expected.to eql author_url }
+
+        it { expect(subject.author_url).to eql(author_url) }
+
+        it 'can be called twice and receive the same value' do
+          expect(subject.author_url).to eq(subject.author_url)
+        end
```

Comment:
Note: tested without my change and it fails.

Reply:
Yes, the test setup is in bad shape. Sorry about that, I haven't used/touch the codebase for a really long time.

Reply:
no worries! This was a long time ago and the gem still works great 👍